server/src/ScriptRunner/Scripts/index/add_new_conty_transition_archive.sql

```diff
@@ -0,0 +1,11 @@
+INSERT INTO public.counties(
+display_name, id, user_admin_id, district_id, self_investigator_id, is_displayed)
+VALUES ('ארכיון מעברים', '9992', 'admin.group9999', '9999', 'self_investigator.group9999',true);
```

Comment:
You should add 'on conflict do nothing' so that the query won't failed if it runs multiple times